actor/v7action/service_instance_sharing_test.go

```diff
@@ -273,5 +275,68 @@ var _ = Describe("Service Instance Sharing", func() {
 
 		itValidatesParameters()
 
+		When("a successful unshare request is made", func() {
+			expectedSpaceGUID := "fake-space-guid"
+			expectedServiceInstanceGUID := "fake-service-instance-guid"
+
+			BeforeEach(func() {
+				fakeCloudControllerClient.GetSpacesReturns(
+					[]resources.Space{{GUID: expectedSpaceGUID}},
+					ccv3.IncludedResources{},
+					ccv3.Warnings{"some-space-warning"},
+					nil,
+				)
+
+				fakeCloudControllerClient.GetServiceInstanceByNameAndSpaceReturns(
+					resources.ServiceInstance{GUID: expectedServiceInstanceGUID},
+					ccv3.IncludedResources{},
+					ccv3.Warnings{"some-service-instance-warning"},
+					nil,
+				)
+			})
+
+			It("makes a request to the cloud controller", func() {
```

Comment:
This is ok, but a way that I prefer to see tests is:
```go
It("makes the right request", ...)

When("the request is successful", func() {
  It("returns warnings and no error", ...)
})

When("the request fails", func() {
  It("returns warnings and an error", ...)
})
```

This is because the first test happens regardless of the response.